packages/host/app/components/matrix/room.gts

```diff
@@ -34,9 +36,35 @@ interface RoomArgs {
     roomId: string;
   };
 }
+
+interface Patch {
+  Args: {
+    command: any;
+  };
+}
+
+class PatchMessage extends Component<Patch> {
+  @service private declare operatorModeStateService: OperatorModeStateService;
+  <template>
+    <Button
+      data-test-command-apply
+      {{on 'click' (perform this.patchCard)}}
+      @loading={{this.patchCard.isRunning}}
+      @disabled={{this.patchCard.isRunning}}
+    >Apply</Button>
+  </template>
+
+  private patchCard = task({ drop: true }, async () => {
+    let commandParsed = this.args.command;
+    let id = commandParsed.id;
+    let attributes = commandParsed.patch.attributes;
+    await this.operatorModeStateService.patchCard.perform(id, attributes);
+  });
+}
```

Comment:
why is this a component? it looks like we are just wrapping a button--no need to make a separate component just for that--just add the button to the template for the room and let's drop the additional component boilerplate.